contracts/v2/paranets/ParanetNeuroIncentivesPool.sol

```diff
@@ -220,15 +251,25 @@ contract ParanetIncentivesPool {
             revert ParanetErrors.NoKnowledgeMinerRewardAvailable(parentParanetId, msg.sender);
         }
 
+        // Updating the Unrewarded TRAC variable in the Knowledge Miner Profile
+        // If limit for reward wasn't exceeded, we set Unrewarded TRAC to 0, otherwise we need to calculate
```

Comment:
can we have a condition when we simply set to 0, so we don't do the entire computation in line 270 ?